website/integrations/services/gatus/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gatus
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Gatus?
+
+> Gatus is a free and open source project for endpoint monitoring. It allows many types of monitoring from pings or http requests to DNS checking and certification expiration. This is all done through yaml files.
+>
+> -- https://github.com/TwiN/gatus
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `gatus.company` is the FQDN of the Gatus install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+Create an OIDC provider with the following settings:
+
+-   Name: 'gatus'
+-   Redirect URL: 'https://gatus.company/authorization-code/callback'
+
+Everything else is up to you and what you want, just don't forget to grab the client ID and secret!
+
+## Gatus configuration
+
+In the config.yml file of Gatus, add the following:
```

Comment:
```suggestion
In the `config.yaml` file of Gatus, add the following:
```

Reply:
@Redlonghead is the file extension `.yml` or `.yaml` or do we know?

Reply:
It works with either extension I just use .yml